randovania/data/json_data/prime3.txt

```diff
@@ -4062,14 +4954,37 @@ Asset id: 9997664456677642630
   * Normal Door to Transit Tunnel P70/Dock to Transit Station Leviathan
   > Event - Pirate Commander
       Trivial
+  > Samus Ship
+      Suit Type ≥ 4 and After Pirate Commander
 
 > Event - Pirate Commander; Heals? False
+  * Event Pirate Commander
+  > Dock to Transit Tunnel P70
+      Trivial
+
+> Event - Destroy Door; Heals? False
+  * Event Transit Station Leviathan Door
+  > Samus Ship
+      Trivial
+
+> Samus Ship; Heals? True
+  > Dock to Transit Tunnel P70
+      Suit Type ≥ 4
+  > Event - Destroy Door
+      Ship Missile
 
 ----------------
 Leviathan
 Asset id: 5604040344596242484
 > Dock to Leviathan Access Portal; Heals? False
   * Normal Door to Leviathan Access Portal/Dock to Leviathan
+  > Event - Control Leviathan Battleship
+      After Valhalla Levithan Code
+
+> Event - Control Leviathan Battleship; Heals? False
+  * Event Control Leviathan Battleship
+  > Dock to Leviathan Access Portal
+      Trivial
 
 ----------------
 Lift Hub
```

Comment:
Lift Hub needs a new event for the bomb slot gate. its only accessible (no tricks) elevator side after its used once on the transit station side. 
- Connect Transit Station door to the event with bombs and suit type > 4
- Connect event to elevator with suit type > 4
- Connect elevator to transit station with suit type > 4 and after gate event